src/Nethermind/Nethermind.State/PartialStorageProviderBase.cs

```diff
@@ -21,34 +27,37 @@ internal abstract class PartialStorageProviderBase
         protected readonly ILogger _logger;
         protected readonly List<Change> _changes = new(Resettable.StartCapacity);
         private readonly List<Change> _keptInCache = new();
+        protected readonly StorageValueMap _map;
```

Comment:
Can add a comment on why this is done instead of `StorageValue` directly in `Change`?

Reply:
Added some comment here as well as in the map itself.